packages/calling/src/Contacts/ContactsClient.ts

```diff
@@ -252,12 +256,13 @@ export class ContactsClient implements IContacts {
   /**
    * Fetches contacts from DSS.
    */
-  private async fetchContactFromDSS(contactsDataMap: ContactIdContactInfo): Promise<Contact[]> {
-    const contactList = [];
-    const dssResult = await this.webex.internal.dss.lookup({ids: Object.keys(contactsDataMap)});
+  private async fetchContactFromDSS(
+    contactsDataMap: ContactIdContactInfo,
+    id: string
```

Comment:
Q. Is this ID different from `contactId`?

Reply:
no it's not. Just called it `id` to not conflict with `contactId` that appears later in the function.